src/utils/parser.js

```diff
@@ -0,0 +1,2252 @@
+                            <p>L’archipel passe sous contrôle Britannique entre 1759 et 1763, puis de nouveau en 1794, avec l’appui des planteurs cherchant à se soustraire à l’abolition décrétée par la Convention.</p>\n        
+                            <p>L’invasion britannique est repoussée par les forces de l’île, composées, entre autres, de plusieurs milliers de nouveaux affranchis et dirigées par Victor Hugues, commissaire civil envoyé par la Convention pour faire appliquer le décret de l’abolition. Celle-ci n’est toutefois promulguée que de manière symbolique, l’esclavage étant remplacé par une forme de travail forcé. </p>\n        
+                            <p>En 1802, une importante armée insurrectionnelle commandée par Louis Delgrès, homme de couleur libre, s’oppose au général Richepanse envoyé rétablir l’esclavage sur ordre de Napoléon Bonaparte, alors Premier Consul. L’insurrection brutalement réprimée s’achève par le suicide de Delgrès et ses compagnons. </p>\n        
+                            <p>Comme la Martinique, la Guadeloupe passe de nouveau sous contrôle Britannique pendant les guerres napoléoniennes. Brièvement cédée à la Suède lors du traité de Paris de 1814, elle revient définitivement sous contrôle français en 1815. </p>\n        
+                            <p>À l’époque de la fondation de la 
+                                <title>Revue des Colonies</title>, les libres de couleur y sont aussi nombreux que les blancs, et possèdent environ un quart des terres. La 
+                                <title>Revue </title>accorde une place importante à l’actualité de la Guadeloupe, sans doute avec le concours de Mondésir Richard, « mandataire des libres de couleur de Guadeloupe » et membre très probable de la « Société d’hommes de couleur » qui fonde le périodique. 
+                            </p>\n        
+                            <bibl>Marcel Dorigny et Bernard Gainot. 
+                                <title>Atlas des esclavages. Traités, sociétés coloniales, abolitions, de l’Antiquité à nos jours</title>. Paris : Éditions Autrement, 2006
+                            </bibl>\n        
+                            <bibl>Frédéric Régent, « Préjugé de couleur, esclavage et citoyennetés dans les colonies françaises (1789-1848) », 
+                                <title>La Révolution française</title>, no. 9 (2015), 
+                                <ref target=\"https://doi.org/10.4000/lrf.1403\">https://doi.org/10.4000/lrf.1403</ref>.
+                            </bibl>\n        
+                            <bibl>Fallope, Josette. 
+                                <title>Esclaves et citoyens. Les Noirs à la Guadeloupe au XIXe siècl
```

Comment:
use `const` instead of `let` for these variables that you don't need to change. Arrays can be constants and still be manipulated (but not overwritten!)